packages/tap-i18n/.npm/plugin/tap-i18n-compiler/.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
I wonder why these files showed up 🤔 Any idea @lolimay ?

Reply:
Sorry. I just pulled down the updates from the upstream. I'll revert these changes :)